python/ray/data/aggregate.py

```diff
@@ -8,13 +8,46 @@
 
 
 @PublicAPI(stability="beta")
-class AggregateFn(object):
+class Aggregation:
```

Comment:
```suggestion
class BoundAggregateFn:
```

The only difference is that this is bound to parameters right?

Reply:
Its primary difference is that it's bound to a data subsetting spec (e.g. column selection), with a secondary attribute being that it also has a name for the output column (in the Arrow table case). I think that `BoundAggregateFn` is a better name, I'll make that change!